<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="PlotWidget" name="graph_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="progress_label">
            <property name="text">
             <string>00.00 %</string>
            </property>
            <property name="indent">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progress_bar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLabel" name="start_wave_label">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="pw1_start_line_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="pw2_start_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="pw7_check_box">
            <property name="text">
             <string>Pw7</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="pw1_check_box">
            <property name="text">
             <string>Pw1</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLineEdit" name="pw7_end_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="pw8_start_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="pw4_end_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLineEdit" name="pw6_end_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="pw5_end_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="pw3_end_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="pw1_end_line_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="pw5_check_box">
            <property name="text">
             <string>Pw5</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="pw5_start_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="pw4_start_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="pw7_start_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="pw8_check_box">
            <property name="text">
             <string>Pw8</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="end_wave_label">
            <property name="text">
             <string>End</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLineEdit" name="pw8_end_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="pw4_check_box">
            <property name="text">
             <string>Pw4</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="pw6_start_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="pw3_check_box">
            <property name="text">
             <string>Pw3</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="pw6_check_box">
            <property name="text">
             <string>Pw6</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="pw2_check_box">
            <property name="text">
             <string>Pw2</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="pw3_start_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="pw2_end_line_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="notes_label">
          <property name="text">
           <string>Notes:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="notes_text_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="save_button">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="skip_button">
            <property name="text">
             <string>Skip</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ignore_button">
            <property name="text">
             <string>Ignore</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="right_spacer_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionFeature_Definitions"/>
    <addaction name="actionPlot_Settings"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Plot"/>
    <addaction name="actionView_data"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFeature_Definitions">
   <property name="text">
    <string>Feature Definitions...</string>
   </property>
  </action>
  <action name="actionPlot_Settings">
   <property name="text">
    <string>Plot Settings...</string>
   </property>
  </action>
  <action name="actionSave_Plot">
   <property name="text">
    <string>Save Plot...</string>
   </property>
  </action>
  <action name="actionView_data">
   <property name="text">
    <string>View data...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pw1_check_box</tabstop>
  <tabstop>pw1_start_line_edit</tabstop>
  <tabstop>pw1_end_line_edit</tabstop>
  <tabstop>pw2_check_box</tabstop>
  <tabstop>pw2_start_line_edit</tabstop>
  <tabstop>pw2_end_line_edit</tabstop>
  <tabstop>pw3_check_box</tabstop>
  <tabstop>pw3_start_line_edit</tabstop>
  <tabstop>pw3_end_line_edit</tabstop>
  <tabstop>pw4_check_box</tabstop>
  <tabstop>pw4_start_line_edit</tabstop>
  <tabstop>pw4_end_line_edit</tabstop>
  <tabstop>pw5_check_box</tabstop>
  <tabstop>pw5_start_line_edit</tabstop>
  <tabstop>pw5_end_line_edit</tabstop>
  <tabstop>pw6_check_box</tabstop>
  <tabstop>pw6_start_line_edit</tabstop>
  <tabstop>pw6_end_line_edit</tabstop>
  <tabstop>pw7_check_box</tabstop>
  <tabstop>pw7_start_line_edit</tabstop>
  <tabstop>pw7_end_line_edit</tabstop>
  <tabstop>pw8_check_box</tabstop>
  <tabstop>pw8_start_line_edit</tabstop>
  <tabstop>pw8_end_line_edit</tabstop>
  <tabstop>save_button</tabstop>
  <tabstop>skip_button</tabstop>
  <tabstop>ignore_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pw2_start_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw1_end_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>737</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>798</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw1_end_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw2_start_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw2_end_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw3_start_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>819</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw3_start_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw2_end_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw3_end_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw4_start_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>828</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw4_start_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw3_end_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw5_start_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw4_end_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw4_end_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw5_start_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>831</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw5_end_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw6_start_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>761</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw6_start_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw5_end_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw6_end_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw7_start_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>801</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw7_start_line_edit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>pw6_end_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>721</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw7_end_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw8_start_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>819</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw8_start_line_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pw7_end_line_edit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
